### ISSUE TITLE
Log file access.log is not being rotated

### ISSUE THREAD

**Issue by nvazquez** [apache/cloudstack#3443]
##### ISSUE TYPE
 * Improvement Request

##### COMPONENT NAME
~~~
Logging
~~~

##### CLOUDSTACK VERSION

~~~
4.11.0, 4.11.1, 4.11.2, 4.12
~~~

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### SUMMARY
The log file `access.log` is not being rotated after the transition to a single jar file for management server on 4.11, causing the file to keep growing in size

##### STEPS TO REPRODUCE

~~~
Use management server normally
~~~

##### EXPECTED RESULTS

~~~
/var/log/cloudstack/manangement/access.log file is rotated
~~~

##### ACTUAL RESULTS

~~~
/var/log/cloudstack/manangement/access.log is a single file
~~~

**Comment by DaanHoogland** [comment]
should be fixed by 7e64945b8846d00c1d938084a83af3979b826cf1

**Comment by DaanHoogland** [comment]
working on it @kriegsmanj : #9324
